ui/perf.html

```diff
@@ -35,9 +35,6 @@
                   class="btn btn-view-nav dropdown-toggle">Dashboard
           </button>
           <ul class="dropdown-menu icon-menu" role="menu">
```

Comment:
Does removing the three menu items (below) automatically remove the "Dashboard" menu item? Or will the "Dashboard" menu item still appear but clicking on it won't do anything? We don't want to just leave an empty "Dashboard" top level menu - that should be hidden. Since we are going to add dashboards in the near future under "Dashboard" is there a way you can just specify that that menu item be hidden?

Reply:
The "Dashboard" menu item is still visible, but has no items to show when clicking it. I can hide it, using an Angular directive.